package.json

```diff
@@ -179,7 +180,7 @@
     "coverageDirectory": "./coverage",
     "collectCoverage": true,
     "collectCoverageFrom": [
-      "lib/**/*.js",
+      "dist/**/*.js",
```

Comment:
Does this mean we need to (a) ensure we always run tests against a "built" dist folder, and (b) need to change our tests to point to `dist` and not `lib`?

Reply:
This was an error on my part. I've reverted this, and sent out a new commit clarifying tests